aeron-client/src/main/c/aeronc.h

```diff
@@ -1632,6 +1680,46 @@ int aeron_subscription_async_destination_poll(aeron_async_destination_t *async);
 int aeron_subscription_close(
     aeron_subscription_t *subscription, aeron_notification_t on_close_complete, void *on_close_complete_clientd);
 
+/**
+ * Get all of the local socket addresses for this subscription.  Multiple addresses can occur if this is a
+ * multi-destination subscription.  Addresses will a string representation in numeric form.  IPv6 addresses will be
```

Comment:
The old style of double spacing after a period breaks browser formatting. I had to retrain myself on this.